code/datums/quirks/good.dm

```diff
@@ -30,7 +30,7 @@
 /datum/quirk/drunkhealing
 	name = "Drunken Resilience"
 	desc = "Nothing like a good drink to make you feel on top of the world. Whenever you're drunk, you slowly recover from injuries."
-	value = 8
+	value = 6 /// SKYRAT EDIT - Quirk Rebalance - Original: value = 8
```

Comment:
This should be left as is imo. Healing is healing. It's not necessarily healing on its own, but it's that it can be stacked with other things for a large cumulative effect